components/script/dom/htmlinputelement.rs

```diff
@@ -1034,7 +1034,16 @@ impl HTMLInputElement {
             InputType::Time => {
                 let mut textinput = self.textinput.borrow_mut();
 
-                if ! textinput.single_line_content().is_valid_time_string() {
+                if !textinput.single_line_content().is_valid_time_string() {
+                    *textinput.single_line_content_mut() = "".into();
```

Comment:
I wonder if this is equivalent to just calling `.clear()` on the `DOMString`?

Reply:
let me try

Reply:
NO. We can't do this. It cause error

Reply:
What error?

Reply:
seems the function not work well.
it just return `()` as error

Reply:
@KiChjang  I think we can open a issue try to refactor this.

Reply:
```
error[E0614]: type `()` cannot be dereferenced
    --> components/script/dom/htmlinputelement.rs:1038:21
     |
1038 |                     *textinput.single_line_content_mut().clear();
     |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `script`.
```

Reply:
So, it means you cannot `dereference` the tuple type which is the `()`.
If you just use `textinput.single_line_content_mut().clear();`, will it work?

Reply:
sounds good. let me try.